client/src/components/Landing.style.js

```diff
@@ -0,0 +1,68 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+  background-color: var(--highlight);
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-flow: column wrap;
+`;
+
+export const H1 = styled.h1`
```

Comment:
Mentioned below, but this might be better as an h2, or even some other kind of element, seeing as it is not the actual title, but just an addition to it! @teenie-quaggard any thoughts on what would be the most semantic?